src/ui/src/js/controllers/admin_garden.js

```diff
@@ -53,15 +53,32 @@ export default function adminGardenController(
     loadGardens();
   };
 
-  EventService.addCallback('admin_system', (event) => {
+  EventService.addCallback('admin_garden', (event) => {
     switch (event.name) {
       case 'GARDEN_CREATED':
-        break;
+        $scope.data.push(event.payload);
+        $apply();
```

Comment:
This is broken, generates an error.

Reply:
@hazmat345 Can you verify that you are still getting an error with this?